docs/src/main/asciidoc/_high-availability/load-balancing/Affinity_Cookie.adoc

```diff
@@ -0,0 +1,44 @@
+[[load-balancer-affinity-cookie]]
+= Support for load-balancers relying on affinity cookie
+
+While the Apache family of load-balancers relies on attaching session affinity (routing) information to a `JSESSIONID` cookie or a `jsessionid` path parameter,
+there are other load-balancers that rely on a cookie to guide the session.
+
+In order to configure the cookie name and other properties use the following CLI script:
+
+[source,options="nowrap"]
+----
+/subsystem=undertow/servlet-container=default/setting=affinity-cookie:add(name=SRV)
+----
+
+The affinity can be specified either in the `instance-id` or by providing `jboss.node.name` property to the server.
```

Comment:
here I would add link to https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc#instance-id-or-jvmroute

Reply:
@simkam Well, mod_cluster doesn't currently support cookie-based affinity, so that would be misleading.

Reply:
ok, so perhaps link to https://github.com/wildfly/wildfly/blob/main/docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Undertow.adoc ? ... it took me some time to figure out where it belongs while reading only HA guide

Reply:
@simkam Actually, you are right, reworked this. So the basic idea  we are giving people full context in the undertow section, but in the HA guide it's more use-case oriented. Done.